front/sssscs/src/http/HttpService.ts

```diff
@@ -16,4 +16,18 @@ axiosInstance.interceptors.request.use(
         value.headers.Authorization = `Bearer ${AuthService.getJWTString()}`;
         return value;
     }
-);
\ No newline at end of file
+);
+
+axiosInstance.interceptors.response.use(
+    (response: AxiosResponse<any, any>) => {
+        return response;
+    },
+    (error: any) => {
+        if (error.response.status === 401) {
+            AuthService.removeJWT();      
+            window.location.href = "/login";
+            // Changing GloState?
```

Comment:
This logic could maybe be put in a logout helper function

Reply:
[3a28530](https://github.com/magley/sssscs/pull/40/commits/3a28530c8d075a11529a9ca6c2ec790c39ac6c22)
This is the best I can do because this login logic and the one called when we manually log out are different since we can't rely on anything React-related here.